### PR TITLE
readme updated

### PR DESCRIPTION
**Pull request by darkcoderrises**

### CODE REVIEW

**Comment by somayjain** [comment]
A small suggestion. 
Have a single commit per issue. It would be helpful in keeping the commit history neat.
If you have multiple commits (like here), you can rebase them to combine into a single commit

**Comment by amoghbl1** [comment]
Yup, definitely needs a rebase...
Also, there's a lot more information that you need to add into the readme, make more changes and add more details. Rebase it into one single commit, and then it'll be ready to be merged.

**Comment by darkcoderrises** [comment]
ok

File: README.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
 # osdg-ranker
-OSDG Ranker. A ranking of people based on the amount of Open Source work they've done.
+OSDG Ranker. A ranking of people based on the amount of Open Source work they've done. This project will serve as a incentive for people to start open source and keep contributing to it. This project will fetch data from github with the help of git api and html scrappers, then stores the data in a database. Then the data is displayed onto the website(once it is ready) http://osdg.iiit.ac.in/. Currently the person's contribution in a repo would be evaluated by the ratio of  sum of lines he/she has changed in the repo and total number of lines in it. Finally this ratio will be multiplied by the value of the repo, which may be given by stars or the number of contributors. A persons total contribution be then hence the sum of his contributions in all the projects he has yet worked on. This would be then updated on a regularly basis.
```
Comment (nisargjhaveri) [inline README.md]: > amount of Open Source work they've done

Are we talking about Open Source contributions in general, or contributions to OSDG-IIITH?

Comment (darkcoderrises) [inline README.md]: general

**Comment by ffledgling** [comment]
This commit looks like it's mixing two things -- (1) What this project is and (2) How it goes/ should go about it
(1) Belongs in the README, (2) belongs in the project roadmap. It'd be great if you could split it into two files called `README.mkd` and `TODO.mkd` or `Project-Plan.mkd`

**Comment by amoghbl1** [comment]
Looks good, might need more updates in the future though.

**Pull request merged**